.travis.yml

```diff
@@ -19,14 +19,32 @@ env:
         - COASTLINEDIR="$HOME/gmt-install-dir/coast"
         - PATH="$INSTALLDIR/bin:$PATH"
         - LD_LIBRARY_PATH="$INSTALLDIR/lib:$LD_LIBRARY_PATH"
+        - TRAVIS_EVENT_TYPE=cron
+        - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+            export PYTHON=3.6
+            export CONDA_REQUIREMENTS="ci/requirements.txt"
+            export BUILD_DOCS=true
+          fi
 
 before_install:
     # Install GMT dependencies
     - bash ci/travis-setup.sh
+    - if [ "$BUILD_DOCS" == "true" ]; then
+        # Install GMT documentation dependencies
+        # Get the Fatiando CI scripts
```

Comment:
Travis doesn't like these comments inside blocks. You have to put them out of the `if` or they can't parse the yml file.